feature/channel/src/main/java/com/m3u/feature/channel/ChannelMask.kt

```diff
@@ -542,11 +561,13 @@ private fun MaskCenterButton(
     onPause: () -> Unit,
     onRetry: () -> Unit,
 ) {
-    Box(modifier, contentAlignment = Alignment.Center) {
+    Box(
+        modifier,
+        contentAlignment = Alignment.Center
+    ) {
         when (maskCenterState) {
             Replay, Play, Pause -> {
-                MaskCircleButton(
-                    state = maskState,
+                MaskCircleButton(state = maskState,
```

Comment:
line break.

Reply:
![image](https://github.com/user-attachments/assets/88676a21-0110-45b1-8295-2395668ea80b)
Why i cannot use the reformat feature with these file, when i use it don't move the `state` to the next line? 
I attached the image from my Code Style Settings

Reply:
the idea reformation would make mistakes sometimes.